regulations/static/regulations/js/source/views/header/preamble-head-view.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
```

Comment:
I guess I should clarify, they _can_, but probably shouldn't. Since `PreambleView` already is associated with `#content-wrapper`, it is better practice to make a `PreambleHeadView` to associate with `.content-header`. Also this is similar to the structure set between `RegView`, `HeaderView` and `SubHeadView`.